src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -103,7 +103,16 @@ def __init__(
         self._name = name or ""
         self._version = version
         self._sub_dir = None if sub_dir is None else os.fspath(sub_dir)
-        self._log_graph = log_graph
+        self._log_graph = log_graph and _TENSORBOARD_AVAILABLE
+
+        if log_graph and not _TENSORBOARD_AVAILABLE:
+            m = """You set log_graph=True but Tensorboard is not available. Please run this command to install it:
+
+            pip install tensorboard
```

Comment:
This will not be printed correctly. All the whitespace to the left will appear. I suggest using a regular string.